### DISCUSSION TITLE
FilledButton `bgcolor` animations seems not working

### DISCUSSION THREAD

**Discussion by stabldev** [flet-dev/flet#3725] [Q&A]
### Question

Hi,
I tried adding `animation_duration` to `ft.FilledButton`, but doesnt seem like working.

### Code sample

```python
ft.FilledButton(
    content=ft.Text("Anime"),
    style=ft.ButtonStyle(
        shape=ft.RoundedRectangleBorder(radius=10),
        color={"": ft.colors.WHITE},
        bgcolor={
            "": ft.colors.BLUE_900,
            "hovered": ft.colors.PRIMARY,
        },
        animation_duration=300,
    ),
    on_click=lambda _: page.go("/anime"),
),
```

### Error message

```bash
No error, it just not animating.
```

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Your code is not runnable. Please share some runnable code for testing, then explain the issue detailly.

**Reply by stabldev** [reply]
sorry for the late reply, here is the repo that I'm working on:
https://github.com/coreproject-moe/CoreProject/tree/f73f26c741e9392c061980d30533f3888b2b1dd3/seeder

you can just:
 - clone it
 - `poetry install`
 - `poetry run flet run -d -r ./src/seeder_flet/main.py`

and more specifically here is the code with animation:
https://github.com/coreproject-moe/CoreProject/blob/f73f26c741e9392c061980d30533f3888b2b1dd3/seeder/src/seeder_flet/lib/views/home.py#L20-L32

issue is - like i said - it is just that background color of button not animating on hover and default state.
i hope it would be enough for you, please lmk if you need any more info.
thanks.
